README.md

```diff
@@ -52,23 +52,11 @@ You should see output similar to the following:
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
 
-
-This application is dependent on Trello, an account for that will need to be set up
```

Comment:
It's worth mentioning that the application is now dependent on a mongo DB server hosted in Atlas.

Because the `mongodb+srv` part of your connection string is hardcoded, it doesn't work with e.g. running a single Mongo DB locally.

You could have made the whole connection string a single env var instead of splitting it up into some hardcoded, plus env vars for server hostname/username/password/default database

Reply:
done